egs/mgb5/s5/local/chain/run_tdnn.sh

```diff
@@ -0,0 +1,263 @@
+#!/bin/bash
+
+# Copyright 2017-2018  Johns Hopkins University (author: Daniel Povey)
+#           2017-2018  Yiming Wang
```

Comment:
Add your name and mention from what system it comes

Reply:
change it to symbolic link to tuning/run_tdnn_1a.sh